lib/reflectable_builder.dart

```diff
@@ -32,16 +58,11 @@ Future<BuildResult> reflectableBuild(List<String> arguments) async {
   } else {
     // TODO(eernst) feature: We should support some customization of
     // the settings, e.g., specifying options like `suppress_warnings`.
-    BarbackSettings settings = new BarbackSettings(
-      <String, Object>{"entry_points": arguments, "formatted": true},
-      BarbackMode.DEBUG,
-    );
-    final builder = new TransformerBuilder(
-      new ReflectableTransformer.asPlugin(settings),
-      const <String, List<String>>{
-        '.dart': const ['.reflectable.dart']
-      },
-    );
+    BuilderOptions options = new BuilderOptions(
+        <String, dynamic>{"entry_points": arguments, "formatted": true},
+        isRoot: true);
+    final builder = new ReflectableBuilder(options);
+    // Plus: new ReflectableTransformer.asPlugin(settings),
```

Comment:
What does this comment mean?

Reply:
That comment was just left over from old code. Deleted.